treeherder/webapp/api/push.py

```diff
@@ -175,6 +175,10 @@ def list(self, request, project):
             msg = f"Specified count exceeds api limit: {max_push_count}"
             return Response({"detail": msg}, status=HTTP_400_BAD_REQUEST)
 
+        if count < 0:
```

Comment:
Nitpick: you say 'positive integer' but `0` would get through here.  Is that okay?

Reply:
I did test with `0` and it worked fine returning an empty dataset, I was just trying to remove the 502 error.  I will see with Aryx says in his review- thanks for pointing this out.

Reply:
Please change it to `if count < 1:`